lib/matplotlib/style/core.py

```diff
@@ -65,7 +66,7 @@ def _apply_style(d, warn=True):
     mpl.rcParams.update(_remove_blacklisted_style_params(d, warn=warn))
 
 
-def use(style):
+def use(styles):
```

Comment:
I agree, I did not think about this.
Then, shoud I just adapt to keep the same API?